plugins/k8s-rebase/scripts/k8s-rebase-validate.sh

```
@@ -0,0 +1,643 @@
+      EXTRA_ARGS=""
+      [[ "$MODE" == "test-only" ]] && EXTRA_ARGS="$TEST_ONLY_PKGS $TEST_ONLY_EXTRA"
+      exec $CONTAINER_RT run --rm \
+        --security-opt label=disable \
+        --privileged \
+        $USERNS_FLAG \
+        -v "$REPO_ROOT:$REPO_ROOT" \
+        -v "$(dirname "$SCRIPT_PATH"):$(dirname "$SCRIPT_PATH"):ro" \
+        -w "$REPO_ROOT" \
+        -e K8S_REBASE_IN_CONTAINER=1 \
+        "$GO_IMAGE" \
+        bash "$SCRIPT_PATH" $MODE_FLAG $EXTRA_ARGS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)